book/effects/time.md

```diff
@@ -137,49 +174,74 @@ There is a lot more info about handling times in the README of [`elm/time`][time
 
 ## `subscriptions`
 
+<!--
 Okay, well how should we get our `Time.Posix` though? With a **subscription**!
+-->
+それではどのように`Time.Posix`を取得するのでしょうか？そう**サブスクリプション**です！
 
 ```elm
 subscriptions : Model -> Sub Msg
 subscriptions model =
   Time.every 1000 Tick
 ```
 
+<!--
 We are using the [`Time.every`][every] function here:
+-->
+ここでは、[`Time.every`][every]関数を使っています：
 
 [every]: https://package.elm-lang.org/packages/elm/time/latest/Time#every
 
 ```elm
 every : Float -> (Time.Posix -> msg) -> Sub msg
 ```
-
+<!--
 It takes two arguments:
 
 1. A time interval in milliseconds. We said `1000` which means every second. But we could also say `60 * 1000` for every minute, or `5 * 60 * 1000` for every five minutes.
 2. A function that turns the current time into a `Msg`. So every second, the current time is going to turn into a `Tick <time>` for our `update` function.
 
 That is the basic pattern of any subscription. You give some configuration, and you describe how to produce `Msg` values. Not too bad!
+-->
+この関数は2つの引数を取ります：
 
+1. ミリ秒単位での時間間隔。`1000`と言えば毎秒を意味します。`60 * 1000`とすれば毎分、`5 * 50 * 1000`とすれば5分毎と言う事も出来ます。
+2. 現時刻をある`Msg`に変換する関数、すなわち毎秒ごとに、現時刻が`update`関数のための`Tick <time>`メッセージに変換されていきます。
+
+これがサブスクリプションの基本的なパターンです。何らかの設定を与え、どのように`Msg`の値を生成するか記述します。悪くないでしょ！
 
 ## `Task.perform`
 
+<!--
 Getting `Time.Zone` is a bit trickier. Our program created a **command** with:
+-->
+`Time.Zone`を取得するのはちょっと工夫が必要です。次のような**コマンド**を生成しました：
 
 ```elm
 Task.perform AdjustTimeZone Time.here
 ```
 
+<!--
 Reading through the [`Task`][task] docs is the best way to understand that line. The docs are written to actually explain the new concepts, and I think it would be too much of a digression to include a worse version of that info here. The point is just that we command the runtime to give us the `Time.Zone` wherever the code is running.
+-->
+この行を理解するための最良の方法は、[`Task`][task]のドキュメントの全てに目を通すことです。あなたが読んでいるドキュメントは、実際のところ新しい概念を説明するために書かれており、ここで端折った情報を提供するために脱線すべきでないと考えます。大事な点はランタイムシステムに対して、このコードの実行時に`Time.Zone`を返すように指示しているだけということです。
```

Comment:
このままでもいいかな〜とは思うんですが、「全てに目を通す」と言われると細か〜いあらゆる部分を読めっていう結構な重荷に感じちゃうかもしれません。
`read through` はおそらくざ〜っと全体に目を通しておいてねっていうニュアンスなので、「一通り目を通す」くらいにすると、読者も「お、じゃあちょっと見てみるか」ってなると思います ;)

> あなたが読んでいるドキュメントは、実際のところ新しい概念を説明するために書かれており、ここで端折った情報を提供するために脱線すべきでないと考えます。

"The docs" は全文の `Task` のドキュメントのことを指していると思います。
そうしないと

> I think it would be too much of a digression to include a worse version of that info here.

という後続の文の 「"that info" ってどれ？」ってなってしまうと思います。
実際にこの guide （「あなたが呼んでいるドキュメント」) では実際に (actually) 新しい概念について説明することはせず、簡単な説明にとどめています。

> `Task` についてのドキュメントに、その新しい概念についての実際の説明がありますので、ここでその話を中途半端に端折って説明して脱線するのはやめようと思います。

という文意だと思います。

Reply:
この行を理解するための最良の方法は、[`Task`][task]のドキュメントに一通り目を通すことで、そのドキュメントの中で、`Task`という新しい概念についての実際の説明がなされています。ここで中途半端な説明をして脱線するのはやめようと思いますが、大事な点はランタイムシステムに対して、このコードの実行時に`Time.Zone`を返すように指示しているだけということです。

うーむいかがでしょう？

Reply:
とてもいいと思います！